### DISCUSSION TITLE
Symfony\Bridge\Twig\Extension\AssetExtension Bug

### DISCUSSION THREAD

**Discussion by alichoo** [symfony/symfony#45443] [Q&A]
| Symfony version | 5.3
| PHP version | 7.3.24

Support Question
I havegot this error on dev env please help:
Argument 1 passed to Symfony\Bridge\Twig\Extension\AssetExtension::getAssetUrl() must be of the type string, null given

**Comment by mbabker** [comment]
Something in one of your Twig templates is calling the `asset()` function with a null value, which the PHP code is rejecting because of an incorrect type.  You should have an error page pointing out where the bad call is being made to help you fix this issue.

**Reply by alichoo** [reply]
Thanks a lot for replying . Yes it gives a rwig page but on server it works correctly
